go-controller/pkg/metrics/ovn_db.go

```diff
@@ -348,89 +349,89 @@ func getOvnDbVersionInfo() {
 	}
 }
 
-// ReadinessProbeCommand runs readiness probes against various targets
-var OvnDBExporterCommand = cli.Command{
-	Name:  "ovn-db-exporter",
-	Usage: "",
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "metrics-bind-address",
-			Usage: `The IP address and port for the metrics server to serve on (default ":9476")`,
-		},
-	},
-	Action: func(ctx *cli.Context) error {
-		bindAddress := ctx.String("metrics-bind-address")
-		if bindAddress == "" {
-			bindAddress = "0.0.0.0:9476"
-		}
+func RegisterOvnDBMetrics(clientset *kubernetes.Clientset, k8sNodeName string) {
+	var podExists bool
 
-		if err := util.SetExec(kexec.New()); err != nil {
-			return err
+	err := wait.PollImmediate(1*time.Second, 300*time.Second, func() (bool, error) {
+		var err error
+		podExists, err = checkPodRunsOnGivenNode(clientset, "name=ovn-sbdb", k8sNodeName)
+		if err != nil {
+			return false, nil
+		}
+		if podExists {
+			return true, nil
```

Comment:
if first pod exists, we don't know if the second one exists, is that ok?

Reply:
Yes, that is fine. The ovnkube-db pods have anti-affinity set such that no two pods can run on the same node.